gradle.properties

```diff
@@ -20,6 +20,9 @@ group=org.apache.kafka
 #  - tests/kafkatest/__init__.py
 #  - tests/kafkatest/version.py (variable DEV_VERSION)
 #  - kafka-merge-pr.py
+#  - streams/quickstart/pom.xml
```

Comment:
Looks like https://github.com/apache/kafka/blob/trunk/streams/quickstart/pom.xml wasn't updated in this PR.